rds-postgres/admin-login/main.tf

```diff
@@ -77,6 +78,10 @@ data "aws_db_instance" "this" {
   db_instance_identifier = var.identifier
 }
 
+data "aws_db_instance" "replica" {
+  db_instance_identifier = var.replica_identifier
```

Comment:
You'll need to use a `count` here to make sure this doesn't fail for databases without a replica.

https://blog.gruntwork.io/terraform-tips-tricks-loops-if-statements-and-gotchas-f739bbae55f9#0223